sqlserver/datadog_checks/sqlserver/activity.py

```diff
@@ -286,20 +286,22 @@ def _obfuscate_and_sanitize_row(self, row):
             # sqlserver doesn't have a boolean data type so convert integer to boolean
             comments, row['is_proc'], procedure_name = extract_sql_comments_and_procedure_name(row['text'])
             if row['is_proc'] and 'text' in row:
+                procedure_content = None
```

Comment:
why we need to define `procedure_content`? i think adding `row['procedure_signature'] = '__procedure_obfuscation_error__'` to exception is enough.

Reply:
Yeah, I just wanted to isolate the obfuscation by itself, but you're right. Updated.